### DISCUSSION TITLE
default geometry and parameters option in pybamm

### DISCUSSION THREAD

**Discussion by Thiyaga2025** [pybamm-team/PyBaMM#4070] [General]
hi all, i observed 'default' geometry and parameters option in pybamm model settings. my question is, when i choose "default" , which parameter and geometry library will be selected by pybamm. how to see the values.
thanks for your help and time.

**Comment by valentinsulzer** [comment]
What specifically do you want to see beyond `print(model.default_parameter_values)` and `print(model.default_geometry)` ?

**Reply by Thiyaga2025** [reply]
hi Valentinsulzer, thanks a lot for the reply. yeah, I can print and check the default parameter and geometry values now. primarily, i was wondering when to use default parameter/geometry option (against the available chemistry libraries like , Chen/Ecker). should i use it as a template and update the parameters as needed.

**Reply by valentinsulzer** [reply]
There's almost never a need to change the default geometry. For parameters, pick whichever one is closest to your cell, and use that as a template

**Reply by Thiyaga2025** [reply]
thanks. this helps.

**Reply by Thiyaga2025** [reply]
@valentinsulzer , this question may sound very novice. but the geometry class features like, span of electrodes and particles are there in parameters library( positive/negative electrode thickness, particle radius) also. what difference the geometry module makes in model setup.
thanks.
